jiant/models.py

```diff
@@ -860,7 +877,7 @@ def _nli_diagnostic_forward(self, batch, task, predict):
 
         # embed the sentence
         classifier = self._get_classifier(task)
-        if self.use_pytorch_transformers:
+        if self.use_pair_embedding:
```

Comment:
I check the papers. All but TransformerXL have pair embedding mode. Huggingface provided apply_boundary_tokens for BERT, XLNet, XLM, RoBERTa, and our code fully implemented their methods. The other three are a bit tricky, GPT and GPT-2 have different boundary function and boundary tokens in LM mode and classification mode, they also have an unique forward function for similarity tasks, leveraging symmetry of similarity task to compensate their single-directness. Transformer-XL in their paper were only experimented with LM task, but since it's an enhancement of GPT, we may borrow GPT's mechanism for classification tasks, if we want so.